src/Symfony/Component/DependencyInjection/Tests/Compiler/ServiceLocatorTagPassTest.php

```diff
@@ -21,10 +21,10 @@
 use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
 use Symfony\Component\DependencyInjection\Reference;
 use Symfony\Component\DependencyInjection\ServiceLocator;
-use Symfony\Component\DependencyInjection\Tests\Fixtures\CustomDefinition;
 use Symfony\Component\DependencyInjection\Tests\Fixtures\TestDefinition1;
 use Symfony\Component\DependencyInjection\Tests\Fixtures\TestDefinition2;
 
+require_once __DIR__.'/../Fixtures/includes/autowiring_classes.php';
```

Comment:
Since I was also in the middle of trying to figure this out, I know why you did this, but probably most people reading this PR will not. I explained the issue [here](https://github.com/symfony/symfony/pull/42347/files/499df3d805f315b5beff767a7a60b2235f8743cf#diff-2d095e6b8882e9c562ea8541b1c0c995b8fd438946e73b4bd4a74f9895a47366) and @chalasr has taken a [different approach](https://github.com/symfony/symfony/pull/48654) to avoid referencing the unloaded class.

However, I think we need to take further action with respect to this bug, because this is indicative that we have tests with hidden dependencies. Relying on the happenstance deterministic (alphabetical) ordering of tests is bad because it means tests cannot be run in isolation, which makes development harder. The best way I know to remedy this is to apply `executionOrder="random"` to the `phpunit.xml` config, which should be done for all such files in this repository.